runtime/doc/options.txt

```diff
@@ -5007,6 +5007,10 @@ A jump table for the options with a short description can be found at |Q_op|.
 	  trail:c	Character to show for trailing spaces.  When omitted,
 			trailing spaces are blank.  Overrides the "space"
 			setting for trailing spaces.
+							*lcs-lead*
+	  lead.c	Character to show for leading spaces. When omitted,
```

Comment:
It looks like it should be lead:c instead of lead.c

Reply:
I have fixed this and pushed this to master. This is my first pull request, and to be honest I wasn't aware that it tracks my master branch. I used a force push to keep the history condensed into a single commit.

I hope this doesn't break things? If things break please let me know and I'll see if I can fix it. I should probably just do another commit next time...